AutoRest/Generators/NodeJS/NodeJS.Tests/Expected/AcceptanceTests/BodyDuration/autoRestDurationTestService.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ * 
+ * Code generated by Microsoft (R) AutoRest Code Generator 0.11.0.0
+ * Changes may cause incorrect behavior and will be lost if the code is
+ * regenerated.
+ */
+
+/* jshint latedef:false */
+/* jshint forin:false */
+/* jshint noempty:false */
+
+'use strict';
+
+var util = require('util');
+var msRest = require('ms-rest');
+var ServiceClient = msRest.ServiceClient;
+var WebResource = msRest.WebResource;
+
+var models = require('./models');
+var operations = require('./operations');
+var moment = require('moment');    
```

Comment:
i don't see we are using the "mement" at service level. I assume this should only be imported at operation/mode related code files